zombienet/0001-basic-warp-sync/0001-generate-warp-sync-database.toml

```diff
@@ -0,0 +1,30 @@
+[relaychain]
+default_image = "docker.io/paritypr/polkadot-debug:master"
+default_command = "/home/miszka/parity/05-zombienet-warp-sync/substrate-wrap.sh"
```

Comment:
I think this one slipped through

Reply:
thx, done in [6ac5d72](https://github.com/paritytech/substrate/pull/12675/commits/6ac5d728f5217c81102de629346f2a16c4c77a5c)